bin/jsize

```diff
@@ -7,61 +7,119 @@ const pkg = require('../package.json')
 updateNotifier({ pkg: pkg }).notify()
 
 const chalk = require('chalk')
-const padEnd = require('pad-end')
+const spinner = require('ora')()
 const program = require('commander')
+const logUpdate = require('log-update')
 const prettyBytes = require('pretty-bytes')
-const spinner = require('ora')('Starting...').start()
+const stringWidth = require('string-width')
+const createTable = require('text-table')
 const jsize = require('../index')
+const originalLog = console.log
+const padding = [' ']
 
 // Parse cli options.
 program
   .usage('[options] <name ...>')
   .option('-v, --verbose', 'Display initial, minified and gzipped size.')
   .parse(process.argv)
 
-// Calucate the padding based on the longest package name.
-const padding = 4 + program.args.reduce(maxLength, 0)
+// Store results for rendering later.
+const bundles = []
+const headings = ['.', '.']
```

Comment:
What about `['package', '']`?

Reply:
I think it looks strange with package personally.